src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.cs

```diff
@@ -74,5 +74,12 @@ public void TestFlowLog()
         {
             NetworkResourcesController.NewInstance.RunPsTest("Test-FlowLog");
         }
+
+        [Fact(Skip = "Rerecord tests")]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
1. Have a problem with starting recording session (not specific to my test)
2. Can't record an automated test until the feature is more generally released. 
3. Could we file it as a work to do?
4. Cmdlet was extensively tested manually